### DISCUSSION TITLE
add file tree to PRs

### DISCUSSION THREAD

**Discussion by 1ak31sha** [community/community#9436] [Pull Requests]
add a collapsable side bar that has a file tree of the PRs files. it could be just the PRs files or the whole repos files. you could require a .json file to configure which files show up, and there could be limits to ensure performance. the file struct can be important to how to files work with each other, and being able to see this in tree format can be very nice for certain projects. also makes is easy to jump around. or to quickly see other files in the same directory(such as a react component directory). yes i can check this in my local but its nice to have.

**Comment by oleg-artene-p2a** [comment]
In PR I need the tree of the changed files to get a bird view of the structure and the idea behind the changes.
Now it's a long flat list of files and it's hard to understand.

**Update**:
Found this great browser extension that adds this feature
https://chrome.google.com/webstore/detail/better-pull-request-for-g/nfhdjopbhlggibjlimhdbogflgmbiahc

**Reply by 1ak31sha** [reply]
cool thanks! will check it out

**Comment by atamaniuc** [comment]
Would be nice to have a feature with a visual display of the file tree changes while reviewing Pull Requests.

**Comment by oleg-artene-p2a** [comment]
@1ak31sha This feature was added, you can enable it

![StaticShot_03-03-2022_21-36-04](https://user-images.githubusercontent.com/83657437/156639669-589ecc77-ebcf-4821-8f6d-ac33422ec7eb.png)

**Reply by 1ak31sha** [reply]
no way! thanks you!!!
